navigation/MainTabNavigator.js

```diff
@@ -274,10 +274,26 @@ const tabBar = createBottomTabNavigator({
     }*/
   },
   Tables: {
-    screen: Tables
+    screen: Tables,
+    navigationOptions: ({navigation, screenProps: {t}}) => {
+      if (navigation.state.routes.length > 0) {
+        let tabBarVisible = true
+        navigation.state.routes.map(route => {
+          if (
+            route.routeName === 'Payment' || route.routeName === 'PaymentOrder' || route.routeName === 'CheckoutComplete' || route.routeName === 'SpiltBillScreen' || route.routeName === 'SplitBillByHeadScreen'
```

Comment:
Perhaps change to this style. Is this better?

['Payment', 'PaymentOrder', etc etc].includes(route.routeName)